assets/formProject.json

```diff
@@ -0,0 +1,162 @@
+[
```

Comment:
On this file and the formVolunteer is where the definition of the form happen, any adjustment needed to text or images needed to add can be added here. Also "subtitles can be added" (see step 5 on formVolunteer.json), images can be added (see step 4 on any of the .json)